portal/views/auth.py

```diff
@@ -43,13 +45,430 @@
 from ..models.role import ROLE
 from ..models.user import (
     User,
-    add_authomatic_user,
+    add_user,
     current_user,
     get_user_or_abort,
 )
 
 auth = Blueprint('auth', __name__)
 
+google_blueprint = make_google_blueprint(
+    scope=['profile', 'email'],
+    login_url='/login/google/',
+)
+
+facebook_blueprint = make_facebook_blueprint(
+    scope=['email', 'user_birthday', 'user_gender'],
+    login_url='/login/facebook/',
+)
+
+
+class FlaskDanceProvider:
```

Comment:
Any guidance on how to organize the code in this file would be much appreciated. I just kinda threw stuff in here because I wasn't sure where else to put it. I was considering the below structure but wanted to get some feedback first.

```
/portal
    /views
        /auth
            /consumer <-- Flask-Dance related files
                facebook.py
                google.py
                consumerAuth.py <-- General functions/apis
            /provider <- Flask-OAuthlib (existing code)
                *.py <- intervention/app related stuff
    /models
        */**/*.py <- similar structure here where applicable
```